### PR TITLE
Document how we generate documentation to rspec.github.io

### PR DESCRIPTION
**Pull request by benoittgt**
Dependent:
- https://github.com/rspec/rspec.github.io/pull/137

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -130,3 +130,19 @@ $ bin/yard server --reload
...
 
+### Publish documentation to rspec.github.io
+
+In other documented rspec gems, we use a rake command from rspec-dev
+to generate documentation. `rspec-rails` is no more synced with
+other gems in term of version. If you want to update the documentation
+to [`rspec.github.io`](https://github.com/rspec/rspec.github.io) you
+need to git clone in the parent folder of rspec-rails the `rspec.github.io`
+repository. You also need from [source branch](https://github.com/rspec/rspec.github.io/tree/source)
+the gem that will be needed as yard plugin: [rspec-docs-template](https://github.com/rspec/rspec.github.io#install-rspec-docs-template-yard-plugin).
```
Comment (benoittgt) [inline DEVELOPMENT.md]: Maybe the commands are not needed. We just need to refer to other repo?

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -130,3 +130,19 @@ $ bin/yard server --reload
 
 Then navigate to `localhost:8808` to view the rendered docs.
 
+### Publish documentation to rspec.github.io
```
Comment (pirj) [inline DEVELOPMENT.md]: Does it make sense to mention that rspec.github.io is the same as rspec.info?

File: DEVELOPMENT.md (hunk 1 above)
Comment (pirj) [reply DEVELOPMENT.md]: I'd prefer only commands 😆

File: DEVELOPMENT.md (hunk 2 above)
Comment (benoittgt) [reply DEVELOPMENT.md]: Yep. Good idea

Comment (benoittgt) [reply DEVELOPMENT.md]: https://github.com/rspec/rspec-rails/compare/d48db33e77663114ee58223f96bdd94d939b0836..56f6c597429f8d7319d9ebfff3fb2505fd970dd7

**Pull request merged**